rules/verify-user-email-with-password-reset.md

```diff
@@ -0,0 +1,41 @@
+---
+gallery: true
+short_description: Verify user email with password reset
+---
+## Verify user email with password reset
+
+This rule will set the user's email as verified in the next login sequence after the password is reseted successfully.
+
+```js
+function (user, context, callback) {
+  var request = require('request@2.56.0');
```

Comment:
Question: Is there a reason why this is within the function scope such that it gets invoked every request (yes, `require()` caches, but still?

Also, why is this pinned to a really specific version of `request`?

Reply:
Though it seems to work without require, I wouldn't do it because this code runs in a Webtask sandbox. Explicit require will be safer. But I agree that it shouldn't be necessary to pin a specific version. This is like that in a few rules in this repository, e.g. email linking, but it might be due to some previous issues with the library. I will be removing the versioning. @thameera any idea on this?